debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlGeometryIT.java

```diff
@@ -218,7 +231,7 @@ else if (i == 2) {
         }
     }
 
-    private DatabaseGeoDifferences databaseGeoDifferences(boolean mySql5) {
+    public static DatabaseGeoDifferences databaseGeoDifferences(boolean mySql5) {
```

Comment:
I actually did not change this. This interface is still private, Github shows an outdated commit content here. :shrug: